src/locales/en-US/errors.json

```
@@ -48,6 +48,9 @@
   "validLinkButGhostMessage": "Your link is valid but I couldn't find that message.",
   "invalidMessageLink": "The message link you sent is invalid.",
   "messageContainsNothing": "The message you're trying to move has no content.",
-  "guildHasNoEmoji": "This server has no emoji or all emojis are not available for me.",
-  "guildHasNoRole": "This server has no role to list."
+  "forgotCheckType": "You forgot to give what I'm checking.",
+  "forgotCheckValue": "You forgot to give the value of what I'm checking.",
+  "channelDoesntExist": "That channel doesn't exist or I couldn't find it.",
+  "guildHasNoRole": "This server has no roles to list.",
+  "guildHasNoEmoji": "This server has no emoji or no emojis are available for me."
```

This should be enough
```suggestion
  "guildHasNoEmoji": "This server has no available emojis for me."
```